project/settings.py

```diff
@@ -249,8 +251,10 @@
 
 EMAIL_HOST = os.environ.get('SMTP_SERVER', 'smtp.postmarkapp.com')
 EMAIL_PORT = os.environ.get('SMTP_PORT', 587)
-EMAIL_HOST_USER = os.environ.get('SMTP_USERNAME', None)  # Required, add to Heroku config or .env file
-EMAIL_HOST_PASSWORD = os.environ.get('SMTP_PASSWORD', None)  # Required, add to Heroku config or .env file
+# Required, add to Heroku config or .env file
```

Comment:
But but but I like inline comments

Reply:
I didn't do it it was my auto formatter 🙈